cli/pazuzu/build.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"github.com/urfave/cli"
+	"github.com/zalando-incubator/pazuzu"
+	"fmt"
+	"io/ioutil"
+)
+
+// Fetches and builds features into a docker image.
+func buildFeatures(c *cli.Context) error {
+	config := pazuzu.GetConfig()
+	storageReader, err := pazuzu.GetStorageReader(*config)
+	if err != nil {
+		fmt.Errorf("Error during storage setup:%s", err)
+		return err
```

Comment:
Should be 

```go
return fmt.Errorf("Error during storage setup:%s", err)
```

since `fmt.Errorf` returns an error, but not prints it https://golang.org/pkg/fmt/#Errorf